modules/system/console/fixtures/mix.webpack.js.fixture

```diff
@@ -11,26 +11,26 @@ module.exports = async () => {
 
     mix.listen('init', function (mix) {
         // Setup Winter path aliases
-        mix._api.alias({
+        mix.api.alias({
```

Comment:
What's the reasoning for this change?

Reply:
I was getting errors using `mix:watch` as soon as I introduced Vue compiling. Turns out that `api` is the correct variable to use for this, and `_api` is the internal reference. The `api` variable actually creates the `_api` variable if it doesn't exist, so I believe that perhaps when Vue compiling is used, the Mix initialisation happens earlier.

Reply:
Huh. Is this something that we should fix in a main branch right now?

Reply:
I'd say so, but I've just done it here for now so I can be sure it's working.